### PR TITLE
added recursive-include to fix issue (#40)

### LINKED ISSUE #40: Missing `tableauhyperapi.h` during pip install
When performing a clean install of pantab from pip (pip install tableauhyperapi), the extension module breaks.

Full traceback below:

```
$ pip install pantab
Collecting pantab
  Using cached https://files.pythonhosted.org/packages/e3/82/e3844ce038e52f9927f43f5ab94984f9bbfb6f6dee25ea1fbd6e4b40eeae/pantab-0.1.0.tar.gz
Requirement already satisfied: pandas in /home/akos/akos_venv/lib/python3.6/site-packages (from pantab) (0.25.3)
Requirement already satisfied: numpy>=1.13.3 in /home/akos/akos_venv/lib/python3.6/site-packages (from pandas->pantab) (1.17.4)
Requirement already satisfied: python-dateutil>=2.6.1 in /home/akos/akos_venv/lib/python3.6/site-packages (from pandas->pantab) (2.8.1)
Requirement already satisfied: pytz>=2017.2 in /home/akos/akos_venv/lib/python3.6/site-packages (from pandas->pantab) (2019.3)
Requirement already satisfied: six>=1.5 in /home/akos/akos_venv/lib/python3.6/site-packages (from python-dateutil>=2.6.1->pandas->pantab) (1.13.0)
Building wheels for collected packages: pantab
  Building wheel for pantab (setup.py) ... error
  ERROR: Command errored out with exit status 1:
   command: /home/akos/akos_venv/bin/python3.6 -u -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'/tmp/pip-install-p1jbd85j/pantab/setup.py'"'"'; __file__='"'"'/tmp/pip-install-p1jbd85j/pantab/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' bdist_wheel -d /tmp/pip-wheel-o3f3iwl1 --python-tag cp36
       cwd: /tmp/pip-install-p1jbd85j/pantab/
  Complete output (28 lines):
  running bdist_wheel
  running build
  running build_py
  creating build
  creating build/lib.linux-x86_64-3.6
  creating build/lib.linux-x86_64-3.6/benchmarks
  copying benchmarks/__init__.py -> build/lib.linux-x86_64-3.6/benchmarks
  copying benchmarks/custom.py -> build/lib.linux-x86_64-3.6/benchmarks
  copying benchmarks/benchmarks.py -> build/lib.linux-x86_64-3.6/benchmarks
  creating build/lib.linux-x86_64-3.6/pantab
  copying pantab/__init__.py -> build/lib.linux-x86_64-3.6/pantab
  copying pantab/_writer.py -> build/lib.linux-x86_64-3.6/pantab
  copying pantab/_reader.py -> build/lib.linux-x86_64-3.6/pantab
  copying pantab/_types.py -> build/lib.linux-x86_64-3.6/pantab
  creating build/lib.linux-x86_64-3.6/pantab/tests
  copying pantab/tests/__init__.py -> build/lib.linux-x86_64-3.6/pantab/tests
  copying pantab/tests/test_writer.py -> build/lib.linux-x86_64-3.6/pantab/tests
  copying pantab/tests/conftest.py -> build/lib.linux-x86_64-3.6/pantab/tests
  copying pantab/tests/test_roundtrip.py -> build/lib.linux-x86_64-3.6/pantab/tests
  copying pantab/tests/test_reader.py -> build/lib.linux-x86_64-3.6/pantab/tests
  running build_ext
  building 'libwriter' extension
  creating build/temp.linux-x86_64-3.6
  creating build/temp.linux-x86_64-3.6/pantab
  gcc -pthread -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall -fPIC -I/home/akos/akos_venv/include -I/usr/local/include/python3.6m -c pantab/_writermodule.c -o build/temp.linux-x86_64-3.6/pantab/_writermodule.o
  pantab/_writermodule.c:6:29: fatal error: tableauhyperapi.h: No such file or directory
  compilation terminated.
  error: command 'gcc' failed with exit status 1
  ----------------------------------------
  ERROR: Failed building wheel for pantab```

### PR DESCRIPTION
**Pull request by chillerno1**
Resolves issue for linux install and closes #40. Details in issue.

### CODE REVIEW

File: pantab/newsfragments/40.bug [hunk 1]
```diff
@@ -0,0 +1 @@
+Added "package_data={"": ["*.h"]}" to main setup call so that header files are correctly copied during build.
```
Comment (WillAyd) [inline pantab/newsfragments/40.bug]:
```suggestion
Fixed issue where source installations would error with `fatal error: tableauhyperapi.h: No such file or directory`
```

whatsnew notes should avoid being technical; want to think on terms of what a end user may have experienced

**Comment by chillerno1** [comment]
No worries, fair call. I've moved the *.h inclusion from MANIFEST.in to main setup call as suggested and then tested build works as intended.

```
copying pantab/_writer.py -> pantab-0.1.0/pantab                                                                                                                                          
copying pantab/_writermodule.c -> pantab-0.1.0/pantab                                                                                                                                     
**copying pantab/tableauhyperapi.h -> pantab-0.1.0/pantab**    
```

**Comment by WillAyd** [comment]
Minor nit on whatsnew wording; if you accept via GH UI I think this is good to go

I'll work on a public release today / tomorrow

**Comment by WillAyd** [comment]
Great job @chillerno1 !

**Pull request merged**